content/dstu2/medication-statement.md

```diff
@@ -26,3 +30,107 @@ Search for MedicationStatements that meet supplied query parameters:
 
 <%= headers 200 %>
 <%= json(:dstu2_medication_statement_bundle) %>
+
+## Create
+
+Create new medicationStatement.
+
+    POST /MedicationStatement
+
+### Headers
+
+To successfully POST a medicationStatement, the following headers must be provided. MedicationStatement creation is supported from the closed endpoint only.
```

Comment:
Verify that all of your references to MedicationStatement are cased correctly. I see several 'medicationStatement'-s